helpers/withApollo.ts

```diff
@@ -6,7 +6,7 @@ import { InMemoryCache } from 'apollo-boost'
 const SERVER_URL = process.env.SERVER_URL
 
 const link = createHttpLink({
-  uri: `${SERVER_URL}/graphql`,
+  uri: `${SERVER_URL}`,
```

Comment:
Not a big deal but `SERVER_URL` it's already a string you do not need to use string interpolation